roles/leadership/global-executor.md

```diff
@@ -0,0 +1,25 @@
+# Global executor
+
+The **Global executor** is the community leader, responsible for guiding the direction of the community and achieving its [mission](https://blackpythondevs.github.io/about).
+
+## Current role leader
+
+[Jay Miller](https://github.com/kjaymiller)
+
+## Responsibilities
+
+The following are the duties and responsibilities of the Global executor:
+
+- Coordinate the appointment of leadership members.
+- Facilitate partnerships with outside organizations.
+
```

Comment:
I would add:

- provide consult and insight into regional decisions to the responsibilities